server.js

```diff
@@ -2989,6 +2989,39 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+camp.route(/^\/sourcegraph\/([\s\S]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
We wish to allow for vendor URLs to support extending it as vendors support more pieces of information. Could you add a part related to this particular piece of information offered by Sourcegraph? Perhaps `/sourcegraph/rrc/…`, for "repository reference count"?